src/components/NewGalaxyModal.js

```diff
@@ -150,21 +250,19 @@ export const NewGalaxyModal = _.flow(
 
   return div({ style: styles.drawerContent }, [
     h(TitleBar, {
-      title: Utils.switchCase(viewMode,
-        ['launchWarn', () => h(WarningTitle, ['Launch Galaxy'])],
-        ['deleteWarn', () => 'Delete Cloud Environment for Galaxy'],
-        [Utils.DEFAULT, () => 'Cloud environment']
+      title: Utils.cond(
+        [viewMode === 'launchWarn', () => h(WarningTitle, ['Launch Galaxy'])],
+        [viewMode === 'deleteWarn', () => 'Delete Cloud Environment for Galaxy'],
+        [!!app && (app.status === 'STOPPED'), () => 'Cloud environment is now paused'],
+        [!!app && (app.status === 'STOPPING'), () => 'Cloud environment is now pausing'],
+        [!!app && (app.status === 'STARTING'), () => 'Cloud environment is now starting'],
+        () => 'Cloud environment'
```

Comment:
Hi Brett, Sorry for the late reply I thought I had sent this at the beginning of the week. Would this be similar to how renderEnvironmentWarning works in NewRuntimeModal? 
Also for viewMode = Create/Delete conditions, would this mean that we should also add in the modal titles into the RenderCreateWarning ... etc blocks?

Reply:
Yep, that's the suggestion. Even though every mode has a `TitleBar`, if the contents and behavior differ along the same lines as the body content, then it could be cleaner to push them down into the individual modes. Doing that would consolidate and simplify the conditional logic, at the cost of a slight bit of code bulk in the multiple `TitleBar`s.